pystan/misc.py

```diff
@@ -139,7 +139,7 @@ def _format_number(num, n_signif_figures, max_width):
     """
     if max_width < 6:
         raise NotImplementedError("Guaranteed formatting in fewer than 6 characters not supported.")
-    if math.isnan(num):
+    if math.isnan(num) or math.isinf(num):
```

Comment:
This was failing when `num` was infinite. I don't know if this is the way to deal with this in python.